native.js

```diff
@@ -601,6 +601,15 @@ Native["com/sun/cldc/io/ResourceInputStream.open.(Ljava/lang/String;)Ljava/lang/
         obj = J2ME.newObject(CLASSES.java_lang_Object.klass);
         obj.data = new Uint8Array(data);
         obj.pos = 0;
+
+        // Instead of loading the emoji images at startup, we load them
+        // as soon as MIDlets load them.
+        if (MIDlet.customEmojiImageRegex) {
```

Comment:
I think we should put configuration parameters like this into the *config* parameter, to better isolate the changes that get made to configure the project to run a particular midlet.